nixos/modules/services/misc/autorandr.nix

```diff
@@ -22,19 +22,9 @@ in {
 
     environment.systemPackages = [ pkgs.autorandr ];
 
-    # systemd.unitPackages = [ pkgs.autorandr ];
+    systemd.packages = [ pkgs.autorandr ];
```

Comment:
@Mic92 You will need to manually configure autorandr to make use of it. Default is often created as a symlink to desired profile https://github.com/phillipberndt/autorandr#how-to-use